x-pack/plugins/security_solution/public/flyout/document_details/right/components/response_section.tsx

```diff
@@ -47,6 +47,11 @@ export const ResponseSection = memo(() => {
           id="xpack.securitySolution.flyout.right.response.previewMessage"
           defaultMessage="Response is not available in alert preview."
         />
+      ) : isPreviewMode ? (
+        <FormattedMessage
+          id="xpack.securitySolution.flyout.right.response.openFlyoutMessage"
+          defaultMessage="Open alert details flyout to access response actions."
```

Comment:
one more here where @nastasha-solomon should weigh in

Reply:
I guess if you want to merge this now and do a follow up PR to make the changes that'd be ok...

Reply:
Opened a copy issue for the labels here-> https://github.com/elastic/security-docs/issues/5484. Yes it will be great to merge the 2 feature PRs and do a follow up. I'll ask Nastasha about timing, if it will be quick, I can get it in here.

Reply:
@christineweng pinged you on Slack to discuss the timing.